charts/victoria-metrics-auth/templates/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
this permissions is not needed for vmauth.

Reply:
Thanks @f41gh7, I will remove this in the next commit.